boards/arm/nucleo_l152re/nucleo_l152re_defconfig

```diff
@@ -25,7 +25,7 @@ CONFIG_CLOCK_CONTROL=y
 CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL=y
 # Use HSI source
 CONFIG_CLOCK_STM32_PLL_SRC_HSI=y
-CONFIG_CLOCK_STM32_PLL_DIVISOR=2
+CONFIG_CLOCK_STM32_PLL_DIVISOR=4
```

Comment:
I think you also need to update `CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC` above

Reply:
Sure, thanks! Done